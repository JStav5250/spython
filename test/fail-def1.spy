if false:
	int x = 3

print(x)
