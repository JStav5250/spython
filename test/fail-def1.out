SSyntaxError: cannot explicitly type variable 'x' while in conditional branches!!
